velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -44,6 +44,12 @@ DEFINE_int32(
     100,
     "The number of elements on each generated vector.");
 
+DEFINE_int32(
+    max_num_varargs,
```

Comment:
Shall we use velox_fuzzer_xxx as gflag prefix?

Reply:
I think it should be fine. The fuzzer runs in another process, that is not connected to the main program.